### PR TITLE
Story/bias adjust wind/3055

### PR DESCRIPTION
**Pull request by conbrad**
- [x] Given I've logged in to Morecast 2.0, When the Wind Speed tab is enabled, Then the 5 columns for GDPS Bias, RDPS Bias, HRDPS BIas, NAM Bias, and GFS Bias display values for any rows where the raw model has data available

# Test Links:
[Landing Page](https://wps-pr-3058.apps.silver.devops.gov.bc.ca/)
[MoreCast 2.0](https://wps-pr-3058.apps.silver.devops.gov.bc.ca/morecast-2)
[Percentile Calculator](https://wps-pr-3058.apps.silver.devops.gov.bc.ca/percentile-calculator)
[MoreCast](https://wps-pr-3058.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-3058.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-3058.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-3058.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Auto Spatial Advisory (ASA)](https://wps-pr-3058.apps.silver.devops.gov.bc.ca/auto-spatial-advisory)
[HFI Calculator](https://wps-pr-3058.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

File: web/src/features/moreCast2/components/MoreCast2Column.tsx [hunk 1]
```diff
@@ -259,7 +259,7 @@ export class PrecipForecastField implements ColDefGenerator, ForecastColDefGener
   }
 
   public generateColDefs = (headerName?: string) => {
-    return this.colDefBuilder.generateColDefs(headerName, false)
+    return this.colDefBuilder.generateColDefs(headerName, true)
```
Comment (dgboss) [inline web/src/features/moreCast2/components/MoreCast2Column.tsx:262]: We don't want to display the precip bias adjusted field yet.

File: web/src/features/moreCast2/components/DataGridColumns.tsx [hunk 2]
```diff
@@ -95,15 +95,9 @@ export const columnGroupingModel: GridColumnGroupingModel = [
 // and visibility of our weather parameter tabs
 function columnGroupingModelChildGenerator(weatherParam: string) {
   // For a given weather model, there are tabs present in the datagrid for each WeatherDetermiante except
-  // WeatherDeterminate.NULL. Additioanlly, bias adjusted predictions only exist for temp and rh.
+  // WeatherDeterminate.NULL
   let determinates: WeatherDeterminate[] = []
```
Comment (dgboss) [inline web/src/features/moreCast2/components/DataGridColumns.tsx:99]: I haven't stepped through the code, but I'm not sure this change will be valid if we're not showing the precip and wind direction bias adjusted values in the data grid?

File: api/alembic/versions/b8aa2d38e9e1_add_bias_adj_cols_for_wind.py [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+# revision identifiers, used by Alembic.
+revision = 'b8aa2d38e9e1'
+down_revision = '4916cd5313de'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic ###
+    op.create_index(op.f('ix_morecast_forecast_precip'), 'morecast_forecast', ['precip'], unique=False)
```
[the remark this answers is not in the document]
Comment (conbrad) [reply api/alembic/versions/b8aa2d38e9e1_add_bias_adj_cols_for_wind.py:21]: These were automatically generated by alembic, we could remove them but I think they will be generated again on the next migration.

File: web/src/features/moreCast2/components/MoreCast2Column.tsx [hunk 4]
```diff
@@ -177,7 +177,7 @@ export class WindDirForecastField implements ColDefGenerator, ForecastColDefGene
...
```
[the remark this answers is not in the document]
Comment (conbrad) [reply web/src/features/moreCast2/components/MoreCast2Column.tsx:180]: Yep, makes sense

File: web/src/features/moreCast2/components/DataGridColumns.tsx (hunk 2 above)
Comment (conbrad) [reply web/src/features/moreCast2/components/DataGridColumns.tsx:99]: Yep it still works

**Review by dgboss — APPROVED** [review]
LGTM

File: api/app/weather_models/__init__.py [hunk 5]
```diff
@@ -12,7 +12,7 @@
 
 # Key values on ModelRunGridSubsetPrediction.
 # Wind direction (wdir_tgl_10_b) is handled slightly differently, so not included here.
```
Comment (brettedw) [inline api/app/weather_models/__init__.py:14]: Very minor, but I think this comment is no longer needed, unless `wdir_tgl_10_**b**` is something different

Comment (conbrad) [reply api/app/weather_models/__init__.py:14]: Actually maybe this is why I'm not getting bias adjusted values for wind dir...

**Review by brettedw — APPROVED** [review]

**Pull request merged**